assets/src/edit-story/components/library/panes/media/media3p/karma/mediaFetching.karma.js

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { waitFor } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../../../../../karma/fixture';
+import apiFetcher from '../../../../../../app/media/media3p/api/apiFetcher';
+
+const media = [
+  {
+    name: 'media/unsplash:1',
+    provider: 'UNSPLASH',
+    imageUrls: [
+      {
+        imageName: 'full',
+        url: 'http://www.img.com/1',
+        width: 200,
+        height: 100,
+        mimeType: 'image/png',
```

Comment:
Having just 1 url might clash with #3089. I'm discussing that PR with Pascal to see if those keys ("full", "large" and "web_stories_thumbnail" are actually needed.

Reply:
Updated.